bitnami/kubeapps/templates/apprepository/rbac.yaml

```
@@ -212,7 +219,7 @@ rules:
     verbs:
       - '*'
   - apiGroups:
-      - ""
+      - kubeapps.com
```

Hi @cristi-vlad ,

Did you check [this comment](https://github.com/bitnami/charts/pull/26488#discussion_r1619105758)?

Note that this line breaks the tests as I mentioned.

I would suggest to keep this line as it was.
```suggestion
      - ""
```

Could you please check if with this the issue you want to fix is solved too?

---

Hello. Yes, i applied that change but pipeline still fails. Will apply your proposal and see. Thank you !